transformation/tests/org.jboss.tools.fuse.transformation.editor.tests.integration/src/main/java/org/jboss/tools/fuse/transformation/editor/FuseProject.java

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: JBoss by Red Hat - Initial implementation.
+ *****************************************************************************/
+package org.jboss.tools.fuse.transformation.editor;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.junit.rules.ExternalResource;
+
+public class FuseProject extends ExternalResource {
```

Comment:
why not simply reusing org.fusesource.ide.camel.model.service.core.tests.integration.core.io.FuseProject ?